tests/generate_packageTest.py

```diff
@@ -99,6 +99,22 @@ def test_main(self):
             setup_py = fl.read()
         self.assertIn("Check code for syntactical or semantical", setup_py)
 
+    def test_main_bear_version_prod(self):
+        fake_prod_version = '99.99.99'
+        with patch('bears.generate_package.VERSION', fake_prod_version):
+            main()
+        with open(self.CSS_BEAR_SETUP_PATH) as fl:
+            setup_py = fl.read()
+        self.assertIn(fake_prod_version, setup_py)
+        self.assertNotIn(VERSION, setup_py)
+
+    def test_main_bear_version_dev(self):
```

Comment:
ah wait actually when we do a release, the version will be a production one for one commit which CI will be run against, I think we should rather patch this here to a dev version.